<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Fancy Mobile Number By Sum</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Add Number (,)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="Delimiter">
          <property name="title">
           <string>Delimiter</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QRadioButton" name="optDelimitComa">
                 <property name="text">
                  <string>&quot; ,&quot;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="optDelimitTab">
                 <property name="text">
                  <string>Tab</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="optDelimitNandR">
                 <property name="text">
                  <string>&quot;\n\r&quot;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="optDelimitNonly">
                 <property name="text">
                  <string>&quot;\n&quot;</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QRadioButton" name="optDelimitPipe">
                 <property name="text">
                  <string>&quot;|&quot;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="optDelimitSemicolon">
                 <property name="text">
                  <string>&quot;;&quot;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="optDelimitColon">
                 <property name="text">
                  <string>&quot;:&quot;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="optDelimitSlash">
                 <property name="text">
                  <string>&quot;/&quot;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Filter (by sum)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Sum Value?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="plainText">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Sorting</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="optNone">
           <property name="text">
            <string>None</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optDesc">
           <property name="text">
            <string>Desc</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optAsc">
           <property name="text">
            <string>Asc</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
